docker-images/sw360empty/docker-entrypoint.sh

```diff
@@ -51,10 +51,12 @@
 
 set -e
 
+export TOMCAT=$(basename /opt/sw360/tomcat-*)
```

Comment:
A simple
```
./sw360chores.pl --build -- up
```
fails with:
```
/usr/local/bin/entrypoint.sh: line 68: /opt/sw360/tomcat-*/conf/Catalina/localhost/logs.xml: No such file or directory
```

Reply:
This was due to an unclean build (sw360-liferay.tar.gz was still in place from older version).

Reply:
is this now fixed as suggested in https://github.com/sw360/sw360chores/pull/65#issuecomment-522875417 ?

Reply:
This should be fixed.